src/main/resources/persistence-jdbc.conf

```
@@ -67,9 +67,14 @@ akka.projection.slick {
 }
 
 cqrs-projection {
-  db {
+  clients {
     name = ${CQRS_PROJECTION_DB_NAME}
     connection-string = "mongodb://"${CQRS_PROJECTION_DB_USER}":"${CQRS_PROJECTION_DB_PASSWORD}"@"${CQRS_PROJECTION_DB_HOST}":"${CQRS_PROJECTION_DB_PORT}"/"${CQRS_PROJECTION_DB_NAME}"?replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
     collection-name = ${CQRS_PROJECTION_COLLECTION_NAME}
   }
+  keys {
+    name = ${CQRS_PROJECTION_DB_NAME}
+    connection-string = "mongodb://"${CQRS_PROJECTION_DB_USER}":"${CQRS_PROJECTION_DB_PASSWORD}"@"${CQRS_PROJECTION_DB_HOST}":"${CQRS_PROJECTION_DB_PORT}"/"${CQRS_PROJECTION_DB_NAME}"?replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
+    collection-name = ${KEYS_PROJECTION_COLLECTION_NAME}
+  }
```

Since the read model db is the same, it could simplified as
```suggestion
  db {
    name = ${CQRS_PROJECTION_DB_NAME}
    connection-string = "mongodb://"${CQRS_PROJECTION_DB_USER}":"${CQRS_PROJECTION_DB_PASSWORD}"@"${CQRS_PROJECTION_DB_HOST}":"${CQRS_PROJECTION_DB_PORT}"/"${CQRS_PROJECTION_DB_NAME}"?replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
    clients-collection-name = ${CQRS_CLIENTS_PROJECTION_COLLECTION_NAME}
    keys-collection-name = ${CQRS_KEYS_PROJECTION_COLLECTION_NAME}
  }
```

---

I suggested to @AsterITA to split the config: I was thinking about the new credential described in PIN-3283 but its purpose is to use it for reading data (API gateway context).
I this case we can use the approach suggested by @galales .
👍